src/components/CaseStudies/StaffSpotlight.tsx

```diff
@@ -1,57 +1,56 @@
-interface Author {
+interface Staff {
     image?: string
     name: string
     title: string
 }
 
-export const AuthorBio: React.FunctionComponent<{
+export const StaffSpotlight: React.FunctionComponent<{
     customer: string
     about: string
-    authors: Author[]
-}> = ({ customer, about, authors }) => (
+    staff: Staff[]
+}> = ({ customer, about, staff }) => (
     <div>
-        {authors.length > 1 ? (
-            // Multi author design
+        {staff.length > 1 ? (
+            // Multi staff design
             <div>
                 <div className="tw-mx-auto tw-max-w-screen-sm">
                     <h5 className="mb-4">About {customer}</h5>
                     <p>{about}</p>
                 </div>
 
                 <div className="tw-grid tw-gap-6 tw-grid-cols-1 md:tw-grid-cols-3 tw-mt-16 tw-max-w-screen-lg tw-mx-auto">
-                    {authors.map(a => (
+                    {staff.map(a => (
                         <div key={a.name} className="md:tw-text-center">
                             <img
-                                className="tw-rounded-full max-w-150 tw-inline"
+                                className="tw-rounded-full max-w-150 tw-inline tw-px-0 tw-pb-sm"
                                 width={150}
                                 height={150}
                                 src={a.image}
                                 alt={a.name}
                             />
                             <h3 className="mt-4">{a.name}</h3>
-                            <p className="text-muted">{a.title}</p>
+                            <p className="tw-text-gray-400">{a.title}</p>
                         </div>
                     ))}
                 </div>
             </div>
         ) : (
-            // Single author design
+            // Single staff design
             <div className="tw-flex tw-flex-col md:tw-flex-row">
-                {authors[0].image && (
+                {staff[0].image && (
                     <div className="col-md-3">
```

Comment:
Since Bootstrap columns add their own horizontal outer padding to columns, we want to remove the horizontal padding if we kept this Bootstrap class here. But another suggestion is just removing it altogether instead of overriding it:

```suggestion
                    <div className="tw-mb-sm md:tw-mb-0">
```

Reply:
Fixed in [1bd8387](https://github.com/sourcegraph/about/pull/5706/commits/1bd83878152ad249d5313eeffe4d9b748e0f1d38)